packages/select/src/MenuButton.tsx

```diff
@@ -0,0 +1,290 @@
+  };
+
+  return (
+    <div
+      // eslint-disable-next-line jsx-a11y/role-has-required-aria-props
+      role="combobox"
+      {...ariaProps}
+      tabIndex={disabled ? -1 : 0}
+      ref={refElement}
+      onClick={onClick}
+      onMouseDown={onMouseDown}
+      onKeyDown={onKeyDown}
+      onMouseEnter={onHover}
+      onMouseLeave={offHover}
+      className={cx(
+        menuButtonStyle,
+        css`
+          height: ${sizeSet.height}px;
+          width: ${sizeSet.width}px;
+          font-size: ${sizeSet.text}px;
+          color: ${deselected ? colorSet.text.deselected : colorSet.text.base};
+          border-color: ${open ? colorSet.border.open : colorSet.border.base};
+          background: linear-gradient(
+            180deg,
+            ${colorSet.background.gradientStart.base} 0%,
+            ${colorSet.background.gradientEnd.base} 100%
+          );
+          box-shadow: ${baseBoxShadow};
+          transition: box-shadow 150ms ease-in-out;
+          padding-bottom: 2px; // Counter-balances inset box shadow to re-center text
+
+          @media only screen and (max-width: ${breakpoints.Desktop}px) {
+            height: ${mobileSizeSet.height}px;
+            width: ${mobileSizeSet.width}px;
+            font-size: ${mobileSizeSet.text}px;
+          }
+        `,
+        {
+          [css`
+            background: linear-gradient(
+              180deg,
+              ${colorSet.background.gradientStart.base} 0%,
+              ${colorSet.background.gradientEnd.hovered} 100%
+            );
+            box-shadow: ${baseBoxShadow}, 0 0 0 3px ${colorSet.shadow.hovered};
+          `]: isHovered && !disabled,
+          [css`
+            ${focusStyle};
+
+            background: linear-gradient(
+              180deg,
+              ${colorSet.background.gradientStart.expanded} 0%,
+              ${colorSet.background.gradientEnd.expanded} 100%
+            );
+            padding-bottom: 0;
+          `]: open && !disabled,
+          [css`
+            &:focus {
+              ${focusStyle}
+            }
+          `]: !disabled,
+          [css`
+            cursor: not-allowed;
+            box-shadow: none;
+            background: ${colorSet.background.disabled};
+            color: ${colorSet.text.disabled};
+            padding-bottom: 0;
+          `]: disabled,
+        },
+      )}
+    >
+      <div className={menuButtonContentsStyle}>
+        <input
+          type="hidden"
+          name={name}
+          value={value}
+          readOnly={readOnly}
+          disabled={disabled}
+        />
+
+        <span
+          className={cx(
+            menuButtonTextStyle,
+            css`
+              margin-left: ${sizeSet.option.margin}px;
+
+              @media only screen and (max-width: ${breakpoints.Desktop}px) {
+                margin-left: ${mobileSizeSet.option.margin}px;
+              }
+            `,
+          )}
+        >
+          {text}
+        </span>
+
+        <CaretDownIcon
```

Comment:
We should transition this such that it rotates based on whether the Select is open or not

Reply:
I think Rob and Dave discussed this and decided on keeping it fixed because it would be more complex to also consider cases where the menu opens upwards (not currently implemented).

Reply:
Makes sense